src/test/rustdoc/synthetic_auto/no-redundancy.rs

```diff
@@ -20,7 +20,7 @@ where
 
 // @has no_redundancy/struct.Outer.html
 // @has - '//*[@id="synthetic-implementations-list"]/*[@class="impl"]//*/code' "impl<T> Send for \
-// Outer<T> where T: Copy + Send"
+// Outer<T> where T: Send + Copy"
```

Comment:
Huh, this is strange. cc @GuillaumeGomez

Reply:
Oh, this is a rebase of an old branch, this change is probably not necessary anymore. Removed.